packages/jest-runner/src/runTest.ts

```diff
@@ -218,12 +219,20 @@ async function runTestInternal(
     };
   }
 
+  // if we don't have `compileFunction` on the env, skip coverage
```

Comment:
Where does this property even come from for our own environments?
Depending on how easy it is for a custom environment to fail this check it might be too silent.

Reply:
nvm just read more of the thread, sorry ^^